azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -59,6 +59,7 @@
   private static final String user = "azkaban";
   private static final String password = "azkaban";
   private static final int numConnections = 10;
+  private static final long recentlyFinishedLifetimeMs = 5000;
```

Comment:
nit:
Will the test take 5s to finish? Will it be possible to reduce the time? 
We can mock this out if required.